platform-http-service-framework/src/main/java/org/hypertrace/core/serviceframework/http/StandAloneHttpContainerEnvironment.java

```diff
@@ -0,0 +1,20 @@
+package org.hypertrace.core.serviceframework.http;
+
+import com.typesafe.config.Config;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.hypertrace.core.grpcutils.client.GrpcChannelRegistry;
+import org.hypertrace.core.serviceframework.config.ConfigClient;
+import org.hypertrace.core.serviceframework.spi.PlatformServiceLifecycle;
+
+@AllArgsConstructor
+public class StandAloneHttpContainerEnvironment implements HttpContainerEnvironment {
+  @Getter private final GrpcChannelRegistry channelRegistry;
```

Comment:
Is this for GRPC clients used by the HTTP server?

Reply:
Correct, even though they're _hosting_ HTTP (or more specifically, REST) APIs, they're still communicating downstream to GRPC services.